yaml_edit/lib/src/editor.dart

```diff
@@ -87,7 +87,7 @@ class YamlEditor {
   String _yaml;
 
   /// Root node of YAML AST.
-  YamlNode _contents;
+  late YamlNode _contents;
```

Comment:
If we want to avoid `late` we can change the constructor to:
`YamlEditor._(this._yaml) : _contents = loadYamlNode(_yaml) {` 

And the update `_initialize` to not run `loadYamlNode`... then we would have to patch `_performEdit` which also calls `_initialize`, but that might actually be good because it runs `loadYamlNode` (once on it's own) and once through `_initialize`, so maybe that's good to cleanup.

We can also do this later...